test/xds/xds_client_custom_lb_test.go

```diff
@@ -180,6 +184,66 @@ func (s) TestWrrLocality(t *testing.T) {
 				{Addr: backend3.Address},
 			},
 		},
+		// Sanity check for weighted round robin. Don't need to test super
+		// specific behaviors, as that is covered in unit tests. Set up weighted
+		// round robin as the endpoint picking policy with per RPC load reports
+		// enabled. Due the server not sending trailers with load reports, the
+		// weighted round robin policy should essentially function as round
+		// robin, and thus should have the same distribution as round robin
+		// above.
+		{
+			name: "custom_lb_child_wrr/",
+			wrrLocalityConfiguration: wrrLocality(&v3clientsideweightedroundrobinpb.ClientSideWeightedRoundRobin{
+				EnableOobLoadReport: &wrapperspb.BoolValue{
+					Value: false,
+				},
+				// BlackoutPeriod long enough to cause load report weights to
+				// trigger in the scope of test case, but no load reports
+				// configured anyway.
+				BlackoutPeriod:          &durationpb.Duration{Seconds: 10},
+				WeightExpirationPeriod:  &durationpb.Duration{Seconds: 10},
+				WeightUpdatePeriod:      &durationpb.Duration{Seconds: 1},
+				ErrorUtilizationPenalty: &wrapperspb.FloatValue{Value: 1},
+			}),
+			addressDistributionWant: []resolver.Address{
```

Comment:
This is pretty awkward to write and to read.  Can you make this a struct that has the address & count instead?

Reply:
Done.